src/reference/governance.rst

```diff
@@ -21,21 +21,23 @@ Organizational structure
 
 .. glossary::
 
-    Project Leads
-        The Project Leads are the two co-creators of Nextstrain:
+   Project Leads
+      The Project Leads are the two co-creators of Nextstrain:
 
-          - `Trevor Bedford <https://bedford.io/team/trevor-bedford/>`__
-          - `Richard Neher <https://neherlab.org/richard-neher.html>`__
+      - `Trevor Bedford <https://bedford.io/team/trevor-bedford/>`__
+      - `Richard Neher <https://neherlab.org/richard-neher.html>`__
 
-        The Project Leads have the authority to make all final decisions for the project.
-        In practice, the Project Leads usually choose to defer that authority to the consensus of the :term:`Core Team`.
+      The Project Leads have the authority to make all final decisions for the project.
+      In practice, the Project Leads usually choose to defer that authority to the consensus of the :term:`Core Team`.
 
-    Core Team
-        Active and substantial contributors to Nextstrain including :term:`Project Leads` who have enough context on the project and its components to contribute to decision-making.
+   Core Team
+      Active and substantial contributors to Nextstrain including :term:`Project Leads` who have enough context on the project and its components to contribute to decision-making.
 
-    Community Contributors
-        Occasional contributors including those from `Bedford Lab <https://bedford.io>`__, `Neher Lab <https://neherlab.org>`__, alumni, or other external groups.
-        Influences decision-making but does not directly participate in it.
+      See the `Nextstrain team page <https://nextstrain.org/team>`__ for a full list of members.
```

Comment:
@kimandrews I just realized you're not on the team page but you should be! Can you add yourself using [these instructions](https://github.com/nextstrain/nextstrain.org/blob/master/static-site/README.md#adding-new-team-members)? Let me know if you need help with it